naoqi_apps/nodes/naoqi_tablet.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+#                                                                             
+#  Copyright 2015 Aldebaran                                                   
+#                                                                             
+#  Licensed under the Apache License, Version 2.0 (the "License");            
+#  you may not use this file except in compliance with the License.           
+#  You may obtain a copy of the License at                                    
+#                                                                             
+#      http://www.apache.org/licenses/LICENSE-2.0                             
+#                                                                             
+#  Unless required by applicable law or agreed to in writing, software        
+#  distributed under the License is distributed on an "AS IS" BASIS,          
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   
+#  See the License for the specific language governing permissions and        
+#  limitations under the License.                                             
+#                                                                             
+# 
+import rospy
+from naoqi_driver.naoqi_node import NaoqiNode
+from std_srvs.srv import (
+    EmptyResponse,
+    Empty,)
+from naoqi_bridge_msgs.srv import ( 
+    ShowImageResponse,
+    ShowImage,)
+
+class NaoqiTablet(NaoqiNode):
+    def __init__(self):
+        NaoqiNode.__init__(self, 'naoqi_tablet')
+        self.connectNaoQi()
+        
+        self.image_path = ""
+        self.showImageSrv = rospy.Service("show_image", ShowImage, self.handleShowImageSrv)
+        self.hideImageSrv = rospy.Service("hide_image", Empty, self.handleHideImageSrv)
+        rospy.loginfo("naoqi_tablet initialized")
+    
+    def connectNaoQi(self):
+        rospy.loginfo("Connecting to NaoQi at %s:%d", self.pip, self.pport)
+        self.tabletProxy = self.get_proxy("ALTabletService")
+        if self.tabletProxy is None:
+            exit(1)
+        
+    def handleShowImageSrv(self, req):
+        try:
+            self.image_path = "http://198.18.0.1/apps/img/" + str(req.file_name.data)
```

Comment:
And isn't "html" missing ? 
"http://198.18.0.1/apps/img/html" ? (as your image is in /home/nao/.local/share/PackageManager/apps/img/html/)